plugins/module_utils/foreman_helper.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, argument_spec, **kwargs):
         )
         args.update(argument_spec)
         supports_check_mode = kwargs.pop('supports_check_mode', True)
+        self._aliases = set(alias for arg in args.values() for alias in arg.get('aliases', []))
```

Comment:
You can use set comprehension

```suggestion
        self._aliases = {alias for arg in args.values() for alias in arg.get('aliases', [])}
```